etc/bin/generate.sh

```diff
@@ -102,6 +102,10 @@ thriftrw --no-recurse --plugin=yarpc --out=encoding/thrift/thriftrw-plugin-yarpc
 thriftrw --no-recurse --plugin=yarpc --out=encoding/thrift/thriftrw-plugin-yarpc/internal/tests encoding/thrift/thriftrw-plugin-yarpc/internal/tests/atomic.thrift
 thriftrw --no-recurse --plugin="yarpc --sanitize-tchannel" --out=encoding/thrift/thriftrw-plugin-yarpc/internal/tests encoding/thrift/thriftrw-plugin-yarpc/internal/tests/weather.thrift
 
+thriftrw --no-recurse --plugin=yarpcv2 --out=v2/yarpcthrift/thriftrw-plugin-yarpcv2/internal/tests v2/yarpcthrift/thriftrw-plugin-yarpcv2/internal/tests/common.thrift
+thriftrw --no-recurse --plugin=yarpcv2 --out=v2/yarpcthrift/thriftrw-plugin-yarpcv2/internal/tests v2/yarpcthrift/thriftrw-plugin-yarpcv2/internal/tests/atomic.thrift
+thriftrw --no-recurse --plugin="yarpcv2 --sanitize-tchannel" --out=v2/yarpcthrift/thriftrw-plugin-yarpcv2/internal/tests v2/yarpcthrift/thriftrw-plugin-yarpcv2/internal/tests/weather.thrift
```

Comment:
@abhinav Do you remember why we have generate.sh instead of `go gen` comments littered throughout the codebase?